build/bin/python-collect-prebuilt-wheels.py

```diff
@@ -23,33 +25,39 @@
     help="Target location to save the downloaded wheels of the requirements.txt file",
 )
 parser.add_argument(
-    "--arch", type=str, required=False, help="Architecture of the target platform"
+    "--target-platform", type=str, required=False, help="Architecture of the target platform"
 )
 parser.add_argument(
-    "--add-dependency", type=str, required=True, help="Additional dependencies that are image specific. Use ',' for multiple dependencies"
+    "--python-version", type=str, required=False, help="Specifies if a python version other than one installed on the system should be used to generate the package dependency report"
 )
-args = parser.parse_args()
 
-requirementPath, destination, arch, add_dependency = args.req_file, args.dest, args.arch, args.add_dependency
+# We are keeping --add-dependency and --no-default-crypto for now so that nothing breaks while we do the transition across all the repos which use this script
+# But we won't be using these parameter going forward
+# TODO: REMOVE THESE ARGUMENTS AFTER THE TRANSITION IS COMPLETE.
+parser.add_argument(
+    "--add-dependency", type=str, required=False, help="Additional dependencies that are image specific. Use ',' for multiple dependencies"
+)
 
-ARTIFACTORY_URL = "https://na.artifactory.swg-devops.com/artifactory/wiotp-generic-local/dependencies/wheels/s390x"
-W3_USERNAME, ARTIFACTORY_TOKEN = "y9bbfm866@nomail.relay.ibm.com", os.environ["ARTIFACTORY_TOKEN"]
```

Comment:
I think W3_USERNAME should be an env var in this file too, rather than raw in the code.  If we don't already have the env var I can add it